tests/end2end/features/yankpaste.feature

```diff
@@ -20,6 +20,19 @@ Feature: Yanking and pasting.
         Then the message "Yanked URL to primary selection: http://localhost:(port)/data/title.html" should be shown
         And the primary selection should contain "http://localhost:(port)/data/title.html"
 
+    Scenario: Yanking URLs with ref and UTM data to clipboard
+        When I open http://kikolani.com/blog-post-promotion-ultimate-guide?utm_source=kikolani&utm_medium=320banner&utm_campaign=bpp
+        And I run :yank
+        Then the message "Yanked URL to clipboard: http://kikolani.com/blog-post-promotion-ultimate-guide" should be shown
+        And the clipboard should contain "http://kikolani.com/blog-post-promotion-ultimate-guide"
+
+    Scenario: Yanking URLs to primary selection
```

Comment:
I don't think this test is needed, as your code isn't really involved with the yank target (clipboard/primary selection)